EpiAware/src/EpiInference/EpiInference.jl

```diff
@@ -1,17 +1,18 @@
-
 """
 Module for defining inference methods.
 """
 module EpiInference
 
-using ..EpiAwareBase: AbstractEpiMethod, AbstractEpiOptMethod,
-                      AbstractEpiSamplingMethod
-using Pathfinder: pathfinder, PathfinderResult
-
-using DynamicPPL, DocStringExtensions
+using ..EpiAwareBase: AbstractEpiMethod, AbstractEpiOptMethod, AbstractEpiSamplingMethod
+import ..EpiAwareBase: _apply_method
```

Comment:
why are you using both here? I thought we decided only to use using?

Reply:
`import`ing a function means that each method on `_apply_method` is associated with that function. `using` only does that if you prefix the method definition with the module. I'm happy to do that instead if you prefer.

Here I wanted `_apply_method` in `EpiAwareBase` because its used differently in the inference submodule and the infection submodule.